pkg/backplaneapi/deprecation_test.go

```diff
@@ -0,0 +1,40 @@
+package backplaneapi
+
+import (
```

Comment:
How about to refactor with `Ginkgo BDD Testing Framework` to align with other parts of the bp-cli ?
https://github.com/openshift/backplane-cli/blob/37a6ee99dcd2a1007458afe3d860ac222432a492/pkg/accessrequest/accessRequest_test.go#L12

Reply:
ha! There is a bit of a mix throughout the repository, so I picked the one I preferred. :-P

I've done the refactor in 0e0875ce03ec86e1ff1cb7750734b58395aaf900